packages/backend/src/modules/tracked-txs/modules/liveness/LivenessUpdater.ts

```diff
@@ -28,12 +28,12 @@ export class LivenessUpdater implements TxUpdaterInterface {
     await this.livenessRepository.addMany(transformedTransactions, knexTx)
   }
 
-  async deleteFrom(
+  async deleteFromById(
     id: TrackedTxId,
     untilTimestamp: UnixTime,
```

Comment:
Let's give it a more suitable name like: fromInclusive currently it's very misguiding